<launch>

    <!--Map Server-->
    <arg name="map" default="build4_f20" />
    <node ns="icreate" pkg="map_server" type="map_server" name="map_server" args="$(find miltbot_map)/map/$(arg map).yaml" output="screen">
    </node>

    <include file="$(find icreate_navigation)/launch/icreate_navigation.launch" />

</launch>